### PR TITLE
Core refactoring and cleanup

### PR DESCRIPTION
**Pull request by pzdeb**
This change might particularly break existing functionality and requires bug fixing

### CODE REVIEW

File: src/command_server.py [hunk 1]
```diff
@@ -39,7 +37,7 @@ class StreamCommandServer(BaseCommandServer):
 
     def _init_server(self, host, port):
         self._app = web.Application(loop=self._loop)
-        self._controller = GameController(600, 600, 0, self.notify_clients)
+        self._controller = get_game(600, 600, 0, self.notify_clients)
```
Comment (ntodosiychuk) [inline src/command_server.py]: I can't understand why we init game with inviders count equals zero?

File: src/core/routes.py [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import asyncio
+from functools import wraps
 
 from aiohttp.web_urldispatcher import UrlDispatcher
-from .base import BaseView
-from functools import wraps
+from core.base import BaseView
```
Comment (ntodosiychuk) [inline src/core/routes.py:5]: One small "trolling comment" for @pzdeb ;)
According to Pep 8 importing you should  put a blank line between each group of imports:
- standard library imports
- related third party imports
- local application/library specific imports

In your case it's line number 5.

File: src/simple_commander/__init__.py [hunk 3]
```diff
@@ -0,0 +1 @@
+__author__ = 'Petro'
```
Comment (ntodosiychuk) [inline src/simple_commander/__init__.py]: Good remark)

File: src/simple_commander/main.py [hunk 4]
```diff
@@ -238,13 +240,23 @@ def kill(self, other_unit, units):
         del units[self.id]
 
 
+__game = None
+
+def get_game(height=None, width=None, invaders_count=None, notify_clients=None):
+    global __game
+
+    if not __game and height and width and invaders_count is not None:
```
Comment (ntodosiychuk) [inline src/simple_commander/main.py:256]: In this implementation with global instance of class there is no reason to leave **new** method in GameController.

File: src/simple_commander/main.py [hunk 5]
```diff
@@ -59,10 +58,13 @@ def __init__(self, x, y, angle, bonus, speed, unit_filename, bullet_filename):
         self.stop_change_speed = 'stop'
 
     def response(self, action, **kwargs):
-        controller = GameController(get_old=True)
+        controller = get_game()
+        if not controller:
+            return
         kwargs['id'] = self.id
```
Comment (alive-web) [inline src/simple_commander/main.py]: useless row

**Pull request merged**